packages/yoga/src/AutoComplete/web/AutoComplete.test.jsx

```diff
@@ -65,6 +65,25 @@ describe('<AutoComplete />', () => {
       );
     });
 
+    it('should not filter the options', () => {
+      const { container, getByDisplayValue } = render(
+        <ThemeProvider>
+          <AutoComplete
+            value="secon"
+            options={['first', 'second', 'third']}
+            shouldfilterOptions={false}
+          />
+        </ThemeProvider>,
+      );
+
+      fireEvent.focus(getByDisplayValue('secon'));
```

Comment:
I would like to leave one suggestion about use of fireEvent to interact with DOM elements.

A great alternative and more complete is userEvent, because simulate full user interactions better. For more infos about difference and questions: [Testing library - User Event](https://testing-library.com/docs/user-event/intro)

Reply:
Sounds good to me, @diegomarcuz  🚀